src/components/Task.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Button, Icon } from 'react-native-elements'
+import Colors from '../constants/Colors'
+
+/**
+ * TODO: add tts
+ * Task contains an icon and a button.
+ * @param {string} props.title: The displayed and spoken title
+ * @param {string} props.icon: The icon for the button
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const Task = props => {
```

Comment:
let's make this a generic `RouteButton` with the following props:

- icon (String) - name of the font awesome icon
- title (String) - label to display (and read by tts) - **already translated by parent component**
- route-handler - (Function?) - a way to navigate to a given route by clicking this button